openwisp_monitoring/device/migrations/0009_update_device_status_for_disabled_critical_checks.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.db import migrations
+from swapper import load_model
+
+Check = load_model('check', 'Check')
+Device = load_model('config', 'Device')
+DeviceMonitoring = load_model('device_monitoring', 'DeviceMonitoring')
+
+
+def update_device_status(apps, schema_editor):
+    critical_metrics = settings.OPENWISP_MONITORING_CRITICAL_DEVICE_METRICS
+    for check in Check.objects.filter(is_active=False):
+        if check.object_id and check.metric.name in critical_metrics:
+            device = Device.objects.get(pk=check.object_id)
+            device.monitoring.update_status('unknown')
```

Comment:
The last 3 lines here like a duplication of the logic of the receiver. Just call the same receiver function here, perform any adaptation if needed, let's not duplicate logic.